src/workerd/api/container.c++

```diff
@@ -30,6 +30,30 @@ void Container::start(jsg::Lock& js, jsg::Optional<StartupOptions> maybeOptions)
   }
   req.setEnableInternet(options.enableInternet);
 
+  KJ_IF_SOME(env, options.env) {
+    auto list = req.initEnvironmentVariables(env.fields.size());
+    for (auto i: kj::indices(env.fields)) {
+      auto field = &env.fields[i];
+      JSG_REQUIRE(!field->name.contains("="), Error,
+          "environment variable names cannot include '=' as seen in name of env variable ",
```

Comment:
JavaScript errors should be complete sentences starting with a capital letter. I'd also shorten this a bit:

```suggestion
          "Environment variable names cannot contain '=': ",
```